### DISCUSSION TITLE
Flows - System collections not available CRUD operations

### DISCUSSION THREAD

**Discussion by NirmalAriyathilake** [directus/directus#24929] [Feature Requests]
### Describe the Bug

I want to do CRUD operations on System collections, specifically user collection using Flows.

This issue is a possible duplication of #13858 and related to the discussion #13716 

I'm opening another issue since it is not resolved yet.

### To Reproduce

Try to create a flow with a CRUD operation on a system collection (directus users)

### Directus Version

v10.7.2

### Hosting Strategy

Self-Hosted (Docker Image)

**Comment by robinsonjohn** [comment]
Although an ugly workaround, I can confirm that creating the flow using another collection, then updating that flow’s column to a system collection directly in the database does work.

**Comment by inovosel** [comment]
As a nicer workaround, you can type in the collection name instead of selecting it from the dropdown menu.

<img width="834" alt="Screenshot 2024-12-06 at 15 36 28" src="https://github.com/user-attachments/assets/3ae61ab2-5f3f-4939-bf2f-f63f1d8ac330">

**Comment by rijkvanzanten** [comment]
The reason the system collections aren't shown there is because the operation exclusively uses the `ItemsService` here:

https://github.com/directus/directus/blob/4232dabf0c0c0a159367d24d88d7b14a40d6527b/api/src/operations/item-read/index.ts#L33

That in turn means that things like email duplication checks for `directus_users`, or table creations for `directus_collections` don't kick in.

We created a util to help with this for the GraphQL implementation which I think we might be able to reuse here:

https://github.com/directus/directus/blob/4232dabf0c0c0a159367d24d88d7b14a40d6527b/api/src/services/graphql/index.ts#L1718-L1722

Happy to take a PR for that 🙂 In the meantime you can use the string approach to getting it to work, but just be aware of the potential risks involved
